src/makeInfinityAdder.js

```diff
@@ -1,12 +1,25 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(value) {
+    if (value === undefined) {
```

Comment:
I think !value would be a better check - we would handle more falsy cases

Reply:
In fact !value lets 0 pass as value "value" inside the if block, which is not desired - one of the tests fail:
 _Should work with chains containing zero as argument (1ms)_